### PR TITLE
SAK-49591 T&Q: Show historical changes for questions in question pools

### PR DESCRIPTION
**Pull request by mpellicer**
This needs to be merged after #12210.

This is paired to https://github.com/sakaiproject/sakai-reference/pull/231

### CODE REVIEW

**Comment by jesusmmp** [comment]
Awesome!

**Review by bgarciaentornos — CHANGES_REQUESTED** [review]
Some minor comments, only related to the jira referred on the title

File: samigo/samigo-app/src/webapp/WEB-INF/faces-config.xml [hunk 1]
```diff
@@ -1005,6 +1005,15 @@
       </converter-class>
    </converter>
 
+    <converter>
```
[the remark this answers is not in the document]
Comment (mpellicer) [reply samigo/samigo-app/src/webapp/WEB-INF/faces-config.xml:1008]: It has the correct spacing, all the other items use the wrong one.

image

**Comment by mpellicer** [comment]
Fixed all the comments.

File: samigo/samigo-app/src/webapp/jsf/questionpool/previewQuestion.jsp [hunk 2]
```diff
@@ -95,8 +95,28 @@
...
+</h:dataTable>
+
+<h:panelGroup styleClass="h3text">
+    <h:outputText value="#{questionPoolMessages.t_historical}" />
+</h:panelGroup>
+
+<h:dataTable id="historical" value="#{questionpool.itemBean.get(0).itemData.itemHistorical}" var="historical" styleClass="table table-striped table-bordered">
+    <h:column>
+        <h:outputText value="#{historical.modifiedBy}" >
+            <f:converter converterId="org.sakaiproject.tool.assessment.jsf.converter.UsermameConverter" />
```
Comment (bgarciaentornos) [inline samigo/samigo-app/src/webapp/jsf/questionpool/previewQuestion.jsp:110]:
```suggestion
            <f:converter converterId="org.sakaiproject.tool.assessment.jsf.converter.UsernameConverter" />
```

File: samigo/samigo-app/src/webapp/WEB-INF/faces-config.xml [hunk 3]
```diff
@@ -1005,6 +1005,15 @@
       </converter-class>
    </converter>
 
+   <converter>
+     <converter-id>
+       org.sakaiproject.tool.assessment.jsf.converter.UsermameConverter
```
Comment (bgarciaentornos) [inline samigo/samigo-app/src/webapp/WEB-INF/faces-config.xml:1010]:
```suggestion
       org.sakaiproject.tool.assessment.jsf.converter.UsernameConverter
```

Comment (mpellicer) [reply samigo/samigo-app/src/webapp/WEB-INF/faces-config.xml:1010]: Nah, I'll remove this.

File: samigo/samigo-app/src/webapp/jsf/questionpool/previewQuestion.jsp (hunk 2 above)
Comment (mpellicer) [reply samigo/samigo-app/src/webapp/jsf/questionpool/previewQuestion.jsp:110]: I'll remove this

File: samigo/samigo-app/src/webapp/WEB-INF/faces-config.xml (hunk 3 above)
Comment (mpellicer) [reply samigo/samigo-app/src/webapp/WEB-INF/faces-config.xml]: Voilá https://github.com/sakaiproject/sakai/pull/12213/commits/f2e4a110a3c2416aa8be4dc729d274d5d875fa3d!

**Review by bgarciaentornos — APPROVED** [review]

**Pull request merged**